example_progs.cpp

```diff
@@ -3536,6 +3536,110 @@ prog demosaic_complex() {
   return prg;
 }
 
+prog demosaic_unrolled() {
+  prog prg;
+  prg.compute_unit_file = "demosaic_unrolled_compute.h";
+  prg.name = "demosaic_unrolled";
+
+// Stencil<uint16_t, 64, 64> &hw_input_stencil = arg_0;
+  prg.add_input("hw_input_stencil");
+  prg.buffer_port_widths["hw_input_stencil"] = 16;
+// Stencil<uint8_t, 62, 62, 3> &hw_output_stencil = arg_1;
+  prg.add_output("hw_output_stencil");
+  prg.buffer_port_widths["hw_output_stencil"] = 8;
```

Comment:
Is there a reason that we have port width = 8 for the output? I think I run into a wiring issue in coreIR generation.